config/rbac/manager_role.yaml

```diff
@@ -8,6 +8,8 @@ rules:
   - apps
   resources:
   - daemonsets
+  - deployments
```

Comment:
This file is autogenerated and has rules for deployments and statefulsets already, see lines #50 and #88